File: www/Variant.js
class Variant {
  constructor(variant, I, Alpha, Beta) {
    this.variant = variant;
    this.I = I;
    this.Iold = 0;
    this.Alpha = Alpha;
    this.Beta = Beta;
    this.Array = [];
  }

  getVariant() {
    return this.variant;
  }

  getI() {
    return this.I;
  }

  getIold() {
    return this.Iold;
  }

  getAlpha() {
    return this.Alpha;
  }

  getBeta() {
    return this.Beta;
  }

  getArray() {
    return this.Array;
  }

  setI(I) {
    this.I = I;
  }

  setIold(Iold) {
    this.Iold = Iold;
  }

  setAlpha(Alpha) {
    this.Alpha = Alpha;
  }

  setBeta(Beta) {
    this.Beta = Beta;
  }

  setArray(Array) {
    this.Array = Array;
  }

  addToArray(value) {
    this.Array.push(value);
  }

  toString() {
    return this.variant + " " + this.I + " " + this.Iold + " " + this.Alpha + " " + this.Beta + " " + this.Array;
  }
}

export default Variant;

File: Variant.py
class Variant:

	def __init__(self, variant, I, Alpha, Beta):
		self.variant = variant
		self.I = I
		self.Iold = 0
		self.Alpha = Alpha
		self.Beta = Beta
		self.Array = []

	def getVariant(self):
		return self.variant

	def getI(self):
		return self.I

	def getIold(self):
		return self.Iold
	
	def getAlpha(self):
		return self.Alpha
	
	def getBeta(self):
		return self.Beta

	def getArray(self):
		return self.Array

	def setI(self, I):
		self.I = I

	def setIold(self, Iold):
		self.Iold = Iold
	
	def setAlpha(self, Alpha):
		self.Alpha = Alpha
	
	def setBeta(self, Beta):
		self.Beta = Beta
	
	def setArray(self, Array):
		self.Array = Array
	
	def addToArray(self, element):
		self.Array.append(element)

	def __str__(self):
		return "Variant: " + str(self.variant) + " I: " + str(self.I) + " Iold: " + str(self.Iold) + " Alpha: " + str(self.Alpha) + " Beta: " + str(self.Beta) + " Array: " + str(self.Array)

